packages/features/bookings/lib/create-recurring-booking.ts

```diff
@@ -1,17 +1,10 @@
+import * as fetch from "@calcom/lib/fetch-wrapper";
 import type { AppsStatus } from "@calcom/types/Calendar";
 
-import * as fetch from "@lib/core/http/fetch-wrapper";
-import type { BookingCreateBody, BookingResponse } from "@lib/types/booking";
+import type { RecurringBookingCreateBody, BookingResponse } from "../types";
 
-type ExtendedBookingCreateBody = BookingCreateBody & {
-  noEmail?: boolean;
-  recurringCount?: number;
-  appsStatus?: AppsStatus[] | undefined;
-  allRecurringDates?: string[];
-  currentRecurringIndex?: number;
-};
-
-const createRecurringBooking = async (data: ExtendedBookingCreateBody[]) => {
+// @TODO: Didn't look at the contents of this function in order to not break old booking page.
```

Comment:
Moved to reusable place, contents isn't touched, things have been made a bit easier to share though.